pages/linux/ipset.md

```diff
@@ -0,0 +1,24 @@
+# ipset
+
+> A tool to create IP sets for firewall rules.
+> More information: <https://manned.org/ipset>.
+
+- Create an ip set named `test_set` which will contain ip addresses:
+
+`ipset create {{test_set}} hash:ip`
```

Comment:
I hope I'm understanding this right. Also we commonly use placeholders instead of names like `test` or `test_set`.
```suggestion
- Create an empty IP set which will contain IP addresses:

`ipset create {{set_name}} hash:ip`
```